### ISSUE TITLE
Only the first custom module with using works

### ISSUE THREAD

**Issue by xgdgsc** [timholy/Revise.jl#339]
I have an issue on ubuntu 18.04 atom 1.40.1 julia 1.2.0 Revise 2.1.8. Say I have two modules in files `MyModule1.jl` and `MyModule2.jl` and I write 

```
push!(LOAD_PATH, ".")
using MyModule1
using MyModule2
```

and modify the modules and recall functions. The result is anything I change inside MyModule1 would be revised but nothing I change in MyModule2 is revised. I checked both files are in `Revise.watched_files`. I tried reversing the order of `using`, in one case it swaps the effect, in another case still the same module works. Any suggestions on solving this?

**Comment by timholy** [comment]
Hmm, that's odd. Can you paste the results of `Revise.watched_files`?

This may help:
https://timholy.github.io/Revise.jl/dev/cookbook/

**Comment by xgdgsc** [comment]
![DeepinScreenshot_select-area_20190829071110](https://user-images.githubusercontent.com/1189869/63898596-3d759a80-ca2c-11e9-9ee8-e536eb75978b.png)

**Comment by xgdgsc** [comment]
Also tested on another Ubuntu 16.04 machine, same issue. atom 1.40.1 julia 1.2.0 Revise 2.1.8

**Comment by timholy** [comment]
Hmm. Do the tests pass for you? The reason I ask is there's a [test for what seems to be the same use pattern](https://github.com/timholy/Revise.jl/blob/685f87c7de4efb88e31f9b6c82dc5102ece8e458/test/runtests.jl#L562-L593).

**Comment by xgdgsc** [comment]
I ran test Revise
```
(v1.2) pkg> test Revise
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
   Testing Revise
 Resolving package versions...
 Installed Example ────── v0.5.3
 Installed EponymTuples ─ v0.2.2
    Status `/tmp/jl_f5q68b/Manifest.toml`
  [da1fd8a2] CodeTracking v0.5.8
  [97e2ac4a] EponymTuples v0.2.2
  [7876af07] Example v0.5.3
  [aa1ae85d] JuliaInterpreter v0.6.1
  [6f1432cf] LoweredCodeUtils v0.3.8
  [bac558e1] OrderedCollections v1.1.0
  [295af30f] Revise v2.1.8
  [2a0f44e3] Base64  [`@stdlib/Base64`]
  [ade2ca70] Dates  [`@stdlib/Dates`]
  [8ba89e20] Distributed  [`@stdlib/Distributed`]
  [7b1f6079] FileWatching  [`@stdlib/FileWatching`]
  [b77e0a4c] InteractiveUtils  [`@stdlib/InteractiveUtils`]
  [76f85450] LibGit2  [`@stdlib/LibGit2`]
  [56ddb016] Logging  [`@stdlib/Logging`]
  [d6f4376e] Markdown  [`@stdlib/Markdown`]
  [44cfe95a] Pkg  [`@stdlib/Pkg`]
  [de0858da] Printf  [`@stdlib/Printf`]
  [3fa0cd96] REPL  [`@stdlib/REPL`]
  [9a3f8284] Random  [`@stdlib/Random`]
  [ea8e919c] SHA  [`@stdlib/SHA`]
  [9e88b42a] Serialization  [`@stdlib/Serialization`]
  [6462fe0b] Sockets  [`@stdlib/Sockets`]
  [8dfed614] Test  [`@stdlib/Test`]
  [cf7118a7] UUIDs  [`@stdlib/UUIDs`]
  [4ec0a83e] Unicode  [`@stdlib/Unicode`]
Skipping Base.active_repl
Skipping Base.active_repl_backend
[ Info: tracking Base
┌ Warning: skipping git tests because Revise is not under development
└ @ Main ~/.julia/packages/Revise/JCJHO/test/runtests.jl:1679
[ Info: tracking Pkg
   Cloning default registries into `/tmp/jl_JDZCib`
[ Info: Assigning UUID cbc2e4ab-c953-5db6-b176-7b77cf6f3601 to A2D
 Resolving package versions...
   Cloning default registries into `/tmp/jl_JDZCib`
   Cloning default registries into `/tmp/jl_JDZCib`
  Updating `/tmp/jl_JDZCib/environments/v1.2/Project.toml`
  [cbc2e4ab] + A2D v0.0.0 [`/tmp/jl_JDZCib/A2D`]
  Updating `/tmp/jl_JDZCib/environments/v1.2/Manifest.toml`
  [cbc2e4ab] + A2D v0.0.0 [`/tmp/jl_JDZCib/A2D`]
   Cloning default registries into `/tmp/jl_JDZCib`
[ Info: Assigning UUID cbc2e4ab-c953-5db6-b176-7b77cf6f3601 to A2D
 Resolving package versions...
   Cloning default registries into `/tmp/jl_JDZCib`
  Updating `/tmp/jl_JDZCib/environments/v1.2/Project.toml`
  [cbc2e4ab] ~ A2D v0.0.0 [`/tmp/jl_JDZCib/A2D`] ⇒ v0.0.0 [`/tmp/jl_JDZCib/dev/A2D`]
  Updating `/tmp/jl_JDZCib/environments/v1.2/Manifest.toml`
  [cbc2e4ab] ~ A2D v0.0.0 [`/tmp/jl_JDZCib/A2D`] ⇒ v0.0.0 [`/tmp/jl_JDZCib/dev/A2D`]
   Cloning default registries into `/tmp/jl_JDZCib`
[ Info: Assigning UUID cbc2e4ab-c953-5db6-b176-7b77cf6f3601 to A2D
 Resolving package versions...
   Cloning default registries into `/tmp/jl_JDZCib`
  Updating `/tmp/jl_JDZCib/environments/v1.2/Project.toml`
  [cbc2e4ab] ~ A2D v0.0.0 [`/tmp/jl_JDZCib/dev/A2D`] ⇒ v0.0.0 [`/tmp/jl_JDZCib/A2D`]
  Updating `/tmp/jl_JDZCib/environments/v1.2/Manifest.toml`
  [cbc2e4ab] ~ A2D v0.0.0 [`/tmp/jl_JDZCib/dev/A2D`] ⇒ v0.0.0 [`/tmp/jl_JDZCib/A2D`]
Test Summary: | Pass  Broken  Total
Revise        |  531       1    532
┌ Warning: Replacing docs for `LibGit2.Consts.GIT_CONFIG :: Union{}` in module `LibGit2.Consts`
└ @ Base.Docs docs/Docs.jl:223
 29.228188 seconds (64.88 M allocations: 2.301 GiB, 3.67% gc time)
Test Summary:   | Pass  Total
Base signatures |    2      2
   Testing Revise tests passed 
```
don' t know how to interpret the result.

**Comment by timholy** [comment]
Those tests look good (the two key lines are the ones that follow lines starting with `Test Summary`, and there are no `Fail` or `Error` tests). I have now come up with a test that isolates this, and indeed it seems different from #228.

**Comment by timholy** [comment]
Should be fixed by #343

**Comment by xgdgsc** [comment]
Great. Thanks!
